src/BrazeKit-dev.js

```diff
@@ -14,7 +14,7 @@ window.appboy = require('@braze/web-sdk');
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-var name = 'Appboy',
+var name = 'Braze',
```

Comment:
Not sure what this controls, please confirm if this should / should not be changed

Reply:
This is something internal for us that we check against what is in our database, and it should not be changed, thanks for the question though!

Reply:
Thanks for confirming! I will revert the change shortly